extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -52,26 +65,48 @@ class AmpJWPlayer extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    const el = this.element;
     this.contentid_ = userAssert(
-        (this.element.getAttribute('data-playlist-id') ||
-      this.element.getAttribute('data-media-id')),
+        (el.getAttribute('data-playlist-id') ||
+        el.getAttribute('data-media-id')),
         'Either the data-media-id or the data-playlist-id ' +
-      'attributes must be specified for <amp-jwplayer> %s',
-        this.element);
+        'attributes must be specified for <amp-jwplayer> %s',
+        el);
 
     this.playerid_ = userAssert(
-        this.element.getAttribute('data-player-id'),
+        el.getAttribute('data-player-id'),
         'The data-player-id attribute is required for <amp-jwplayer> %s',
-        this.element);
+        el);
+
+    this.contentSearch_ = el.getAttribute('data-content-search') ||
+        '';
+    this.contentContextual_ = el.getAttribute('data-content-contextual') ||
```

Comment:
Where is `data-content-contextual` and `contentContextual_` used? It looks like we actually use `this.contentSearch_ === '__CONTEXTUAL__'` to determine that the embed is contextual, not this flag.

Reply:
@robwalch @dYale 

Believe this also ties to the comment for line 163 - essentially differentiating search / contextual functionality

Reply:
the flag determines algorithm, search determines value.

`__CONTEXTUAL__` is for the player to listen for so it can replace with context of DOM.